lib/rucio/web/rest/flaskapi/v1/scope.py

```diff
@@ -63,7 +64,7 @@ def get(self):
         :status 406: Not Acceptable
         :returns: :class:`String`
         """
-        return dumps(list_scopes())
+        return dumps(list_scopes(filter={}, vo=request.environ.get('vo')))
```

Comment:
Don't think there's any need for `filter={}` to be here (pedantically trying to avoid clutter)